plugins/filter/junit2obj.py

```diff
@@ -30,9 +30,10 @@
       description: The junit report xml text data
       type: str
       required: true
-    - extra_metadata:
-      description: additional metadata dict added at the root of resulting JSON object
-      type: dict
+    - object:
```

Comment:
Options[1] expect key values, not a list, `xml_report_text` also needs to be changed.
```suggestion
    object:
```

This is required or the documentation won't be rendered.

[1] https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-fields